docs/management/managing-saved-objects.asciidoc

```diff
@@ -1,81 +1,82 @@
-type.
+Saved objects store your visualizations, dashboards, maps, Canvas workpads, and more.
+You can view, edit, delete, import, and export saved objects in 
+*Management > Kibana > Saved Objects*.  
 
 [role="screenshot"]
-image::images/management-saved-objects.png[]
+image::images/management-saved-objects.png[Saved Objects]
 
 [float]
 [[managing-saved-objects-view]]
-=== View, edit, and delete objects
-
-Each object listed in *Saved Objects* has an *Actions* menu with two items:
-
-* *In app* opens the object in its associated application so you can view and 
-edit it.
-* *Relationships* shows other objects that use this object so you know the 
-impact of deleting it.
-
-To delete a saved object, select its checkbox and click *Delete*.
-
-[float]
-[[managing-saved-objects-object-definition]]
-=== Edit an object definition
-
-*Saved Objects* offers an advanced *Edit* page for modifying the object definition. 
-To open the page, click the object name in the list of objects. You can change 
-the object title, add a description, and modify the JSON that defines the 
-object properties.
+=== View, edit, and delete
 
-If you access an object whose index has been deleted, you can:
+* To view and edit an object in its associated application, click the object title.
 
-* Recreate the index so you can continue using the object.
-* Delete the object and recreate it using a different index.
-* Change the index name referenced in the object's `kibanaSavedObjectMeta.searchSourceJSON` to point to an existing
-index pattern. This is useful if the index you were working with has been renamed.
+* To show objects that use this object, so you know the 
+impact of deleting it, click the actions icon image:images/actions_icon.png[Actions icon] 
+and select *Relationships*.
 
-WARNING: Validation is not performed for object properties. Submitting an invalid 
-change will render the object unusable. A more failsafe approach is to use the 
-*Discover*, *Visualize*, or *Dashboard* pages to create new objects instead of 
-directly editing an existing one.
+* To delete one or more objects, select their checkboxes and click *Delete*.
 
 [float]
 [[managing-saved-objects-export-objects]]
-=== Export and import objects
+=== Import and export
 
-In some cases you might want to export and import a set of save objects.
-For example:
+Using the import and export commands, you can move objects between different 
+{kib} instances. This action is useful when you 
+have multiple environments for development and production. 
 
-* You might have multiple environments for development and production. The export 
-and import commands allow you to move objects between different {kib} instances.
-* You have a large number of objects to update and want to batch this process.  
+Import and export also work well when you have a large number 
+of objects to update and want to batch the process.  
 You can export the objects to JSON, make the JSON modification, and re-import 
```

Comment:
I know this part didn't get touched by you but wanted to note it's no longer true. The new export gives NDJSON files that are hard to make modifications to and wouldn't be recommended to be modified. On the other hand, migrations will run on the objects if ever the structure needs to change.